build-a-backend-with-prisma-in-a-typescript-node-project-ca6628d3/notes/04-prisma-use-prisma-studio-to-create-a-record.md

```diff
@@ -0,0 +1,21 @@
+# [Use Prisma Studio to Create a Record](https://egghead.io/lessons/prisma-use-prisma-studio-to-create-a-record)
+
+<TimeStamp start="0:16" end="0:20">
+
+Run `npx prisma studio`
+
+</TimeStamp>
+
+<TimeStamp start="0:22" end="0:27">
+
+Prisma studio is a fully features GUI to look at databases created with Prisma.
+
+
+</TimeStamp>
```

Comment:
An empty line both before closing and after opening TimeStamp is necessary in MDX@1.